msteams-platform/resources/team-chat-member-api-changes.md

```diff
@@ -45,8 +45,8 @@ Here’s a sample JSON representation of what’s returned by these APIs today.
-* We are creating a new API for retrieving the profile information for a single user. We will document this new API in March 2020, but we can provide a few details: it will take the ID of the team/chat and a [UPN](https://docs.microsoft.com/windows/win32/ad/naming-properties#userprincipalname) (`userPrincipalName` above), Azure Active Directory Object ID (`objectId` above), or the Teams user ID (`id` above) as parameters and return the profile information for that user. Note: this API is currently available, but the documentation isn't finished, and the schema of the JSON object it returns will change slightly in the near future: we are changing `objectId` to `aadObjectId` to match what it's called in the `Activity` object of a Bot Framework message.
+* We have created a new paged API [`TeamsInfo.GetPagedMembersAsync`](https://docs.microsoft.com/microsoftteams/platform/bots/how-to/get-teams-context?tabs=dotnet#fetching-the-roster-or-user-profile) for retrieving profile information for members of a chat/team. This API is now available with the Bot Framework 4.8 SDK, for development in all other versions use the [`GetConversationPagedMembers`](https://docs.microsoft.com/dotnet/api/microsoft.bot.connector.conversationsextensions.getconversationpagedmembersasync?view=botbuilder-dotnet-stable) method. Note: In either v3 or v4, the best action is to upgrade to the latest point release (3.30.0 or 4.8 respectively). 
+* We've created a new API [`TeamsInfo.GetMemberAsync`](https://docs.microsoft.com/microsoftteams/platform/bots/how-to/get-teams-context?tabs=dotnet#get-single-member-details) for retrieving the profile information for a single user. It takes the ID of the team/chat and a [UPN](https://docs.microsoft.com/windows/win32/ad/naming-properties#userprincipalname) (`userPrincipalName` above), Azure Active Directory Object ID (`objectId` above), or the Teams user ID (`id` above) as parameters and returns the profile information for that user. Note: we are changing `objectId` to `aadObjectId` to match what it's called in the `Activity` object of a Bot Framework message. The new API is available with the with version 4.8 of the Bot Framework SDK. It will soon be available in Bot Framework 3.x as well, meanwhile you can use the [REST](https://docs.microsoft.com/microsoftteams/platform/bots/how-to/get-teams-context?tabs=json#get-single-member-details) endpoint.
```

Comment:
This should be "It will soon be available in the Teams SDK extension for the Bot Framework 3.x SDK"

Reply:
Good call, updated.